drivers/input/input_stmpe811.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2023 Antmicro <www.antmicro.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define CHIP_ID 0x0811U
+
+/* Touch Screen Pins definition */
+#define STMPE811_GPIO_PIN_4 BIT(4)
+#define STMPE811_GPIO_PIN_5 BIT(5)
+#define STMPE811_GPIO_PIN_6 BIT(6)
+#define STMPE811_GPIO_PIN_7 BIT(7)
```

Comment:
header is not self-contained either. note that all this code can be moved to the source file, since this is only used by a single module.

Reply:
This has been moved to `input_stmpe811.c`.